src/layouts/about/our-team.jsx

```diff
@@ -0,0 +1,17 @@
+// import React from 'react';
```

Comment:
@rgoodie - take a look at gist i made - i have made adjust of how we can handle Team and Members components - https://gist.github.com/meltedspork/05350e1ec8cfda1b7ad16914ec63d103

As it follows React's Proptype - see:  https://reactjs.org/docs/typechecking-with-proptypes.html
and also how we set state in constructor - see: https://reactjs.org/docs/state-and-lifecycle.html

Also for data, if we are doing static data -generally we should be using it as json format in flat file, not JS file to manage data and we can easily import json file and that would be bundled with transpiles.